config/default.yml

```diff
@@ -1474,6 +1474,12 @@ Lint/ElseLayout:
   Enabled: true
   VersionAdded: '0.17'
 
+Lint/EmptyBlock:
+  Description: 'This cop checks for blocks without a body.'
+  Enabled: pending
+  VersionAdded: '0.94'
```

Comment:
The next release will be 1.0 and all currently pending cops will be enabled there by default. I'll ponder if we should merge this one after 1.0.